custom_components/foxess_modbus/entities/modbus_sensor.py

```diff
@@ -63,11 +69,19 @@ def __init__(
     @property
     def native_value(self):
         """Return the value reported by the sensor."""
-        value = original = self._controller.read(self.entity_description.address)
-        if value is None:
-            return None
+        original = 0
```

Comment:
Should we be returning 0 for sensors we can't read? Won't this also set all sensors to 0 on start-up and mess with the history?

Reply:
If any of the reads return None, we still return None (line 76).

This just sets `original` as a clean state, onto which the bits from the individual register reads are OR'ed

Reply:
Missed that, thanks.